core/src/main/java/org/bitcoinj/quorums/GetQuorumRotationInfo.java

```diff
@@ -29,7 +29,6 @@
 
 public class GetQuorumRotationInfo extends Message {
 
-    private long baseBlockHashesCount;
     private ArrayList<Sha256Hash> baseBlockHashes;
```

Comment:
The number of `baseBlockHashes` does not need to be stored in a separate variable because the serialization will include the size of that list.